oecophylla/util/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.0.1"
```

Comment:
Do we really want this?  should the version number go in the main __init__.py?

Reply:
I have no idea how version number inits go. :) Just copied from other directory...